src/wheel/bdist_wheel.py

```diff
@@ -11,21 +11,22 @@
 import re
 from collections import OrderedDict
 from email.generator import Generator
+import distutils
```

Comment:
Move the module level imports to the top of the group.

Reply:
ok